docs/maintainers/repository-structure.md

```diff
@@ -29,6 +29,7 @@ Each track has it's own directory containing a README.md that outlines the conce
   - **practice:** The Practice Exercises for that language.
 - **README.md:** The track's README, which should contain a short description of the goal of the repository as well as the status of the track's v3 transition. A template is provided during the bootstrapping process.
 - **reference:** Files that help maintainers develop the track. These could be reference documents that help explain track-specific concepts for which no appropriate online document could be found, files on language-specific elements of programming concepts, or something else. Maintainers are free to organise this directory in the way they feel it best contributes to developing their track.
+  - **README.md:** The README outlines the concepts that are necessary to learn to become fluent in that language. It could also list any reference documents that have been written.
```

Comment:
@ErikSchierboom: 2 times `README.md`?

Reply:
It is consistent with the description of the readme above, but I agree that we could rephrase both and lose the readme. PR appreciated ;)